app-backend/datamodel/src/main/scala/TileLayerQuality.scala

```diff
@@ -0,0 +1,31 @@
+package com.rasterfoundry.datamodel
+
+import cats.implicits._
+import io.circe._
+
+sealed abstract class TileLayerQuality(val repr: String) {
+  override def toString = repr
+}
+
+object TileLayerQuality {
+  case object NA extends TileLayerQuality("NA")
+  case object Good extends TileLayerQuality("GOOD")
+  case object Better extends TileLayerQuality("BETTER")
+  case object Best extends TileLayerQuality("BEST")
+
+  def fromString(s: String): TileLayerQuality = s.toUpperCase match {
+    case "NA"     => NA
```

Comment:
Are there cases where we'll have a value for tile layer quality, but it will be NA? If not, then probably we should leave off the `NA` enum and just let the optionality of the field indicate missing